src/SuperstateToken.sol

```diff
@@ -107,37 +112,6 @@ contract SuperstateToken is ISuperstateToken, ERC20Upgradeable, PausableUpgradea
         _disableInitializers();
     }
 
-    /**
-     * @notice Initialize the contract
-     * @param _name The token name
-     * @param _symbol The token symbol
-     */
-    function initialize(string calldata _name, string calldata _symbol) public initializer {
-        __ERC20_init(_name, _symbol);
-        __Pausable_init();
-    }
-
-    /**
-     * @notice Initialize version 2 of the contract.
-     * @notice If creating an entirely new contract, the original `initialize` method still needs to be called.
-     */
-    function initializeV2() public reinitializer(2) {
-        // Last usage of `_deprecatedAdmin` variable here.
-        // After this call, owner() is the source of truth for authorization.
-        if (msg.sender != _deprecatedAdmin) revert Unauthorized();
-        __Ownable2Step_init();
-    }
-
-    /**
-     * @notice Initialize version 3 of the contract
-     * @notice If creating an entirely new contract, the original `initialize` method still needs to be called.
-     */
-    function initializeV3(AllowList _allowList) public reinitializer(3) {
```

Comment:
This prevents us from deploying new funds as v4 immediately.

Instead, we'd have to deploy v1, v2, v3 and call initialize functions on each of them. I think this makes sense for USTB/USCC as they've already been deployed.

But for new funds could argue to just deploy right at v4 as we'd have to create new scripts anyways for those new funds.

Reply:
so new initializer for starting at v4?

Reply:
Discussed offline, will keep all initializer functions and new tokens will deploy in order and call all previous intializers